cypress/locators/anchor-navigation/index.js

```diff
@@ -0,0 +1,14 @@
+import {
+  ANCHOR_NAVIGATION,
+  ANCHOR_NAVIGATION_STICKY_NAVIGATION,
+} from './locators';
+
+// component preview locators
+export const anchorNavigation = () => cy.get(ANCHOR_NAVIGATION);
+export const anchorNavigationStickyNavigation = text => anchorNavigation()
+  .find(ANCHOR_NAVIGATION_STICKY_NAVIGATION)
+  .contains(text);
+export const anchorNavigationStickyMainPage = text => anchorNavigation()
+  .find('div')
+  .find('h2')
```

Comment:
I think we can write just only one `find` as `.find('div h2')`

Reply:
updated